### ISSUE TITLE
Moving from Sass-Lint to Stylelint

### ISSUE THREAD

**Issue by evyperez** [oficiodesign/initial-css#1]
We need to switching Sass Lint to stylelint because Sass Lint is no longer being maintained.

**Comment by Eduruiz** [comment]
My last stylelintrc.yml is far from what we get with sass-lint, but it's a start
```
---
plugins:
- stylelint-selector-bem-pattern

extends:
- stylelint-config-sass-guidelines
- stylelint-config-rational-order

rules:
  no-duplicate-selectors: true
  declaration-block-no-duplicate-properties: true
  order/properties-alphabetical-order:
  selector-class-pattern:
  - "^(?:(?:o|c|u|t|s|is|has|_|js|qa)-)?[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*(?:__[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:--[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*)?(?:\\[.+\\])?$"
  - message: Class names should match the hyphenated BEM naming convention
  ```

**Comment by robsonsobral** [comment]
Do you mind replace the alphabetical order with the one used on https://github.com/primer/stylelint-config-primer/blob/master/property-order.js?

**Comment by Eduruiz** [comment]
> Do you mind replace the alphabetical order with the one used on https://github.com/primer/stylelint-config-primer/blob/master/property-order.js?

padding first! :)

**Comment by robsonsobral** [comment]
To [help a little](https://github.com/twbs/bootstrap/blob/ba878eb542ab6c04786741569ba089d02e9bea46/.stylelintrc#L36)

**Comment by Eduruiz** [comment]
wait! are you working on it or this one for me @evyperez?

**Comment by robsonsobral** [comment]
Fixed on 729dd2d0e1bc1306bc6a03c23db3234dc9f3b821
